cmd/gpu_plugin/gpu_plugin.go

```diff
@@ -204,6 +227,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	if opts.sharedDevNum == 1 && opts.resourceManagement {
+		klog.Warning("Trying to use fractional resources with shared-dev-num 1 is pointless")
```

Comment:
```suggestion
		klog.Error("Trying to use fractional resources with shared-dev-num 1 is pointless")
```
Could you please fix the line 226 too? Providing `shared-dev-num` < 1 is a clear error condition from a user's point of view.

Reply:
sure